README.md

```diff
@@ -57,6 +57,7 @@ Real-time electricity data is obtained using [parsers](https://github.com/tmrowc
 - Canada (Nova Scotia): [Nova Scotia Power](http://www.nspower.ca/en/home/about-us/todayspower.aspx)
 - Canada (Ontario): [IESO](http://www.ieso.ca/power-data)
 - Canada (Prince Edward Island): [Government of PEI](http://www.gov.pe.ca/windenergy/chart.php)
+- Canada (Quebec): [HydroQuebec]http://www.hydroquebec.com/reliability-coordinator/
```

Comment:
Move this to the capacity section?

Reply:
I can, but this will be where realtime data comes from (if HQ opens it). See last comment on #166  --> The "Reliability Coordinator" is basically the ISO.

Reply:
Let's add this when we actually have realtime data written in the parser. Before, there's no reason to add it (we're being inconsistent else).
For now, let's remove.